src/hooks/useSoftware/softwareFields.js

```diff
@@ -83,6 +85,7 @@ export const softwareFields = [
     form: {
       control: 'number',
       show: true,
+      validation: Yup.number().min(0),
```

Comment:
Could use a nicer validation message.
```suggestion
      validation: Yup.number().min(0, 'Quantity cannot be below zero.'),
```